app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -8,6 +8,8 @@ def initialize(auction:)
   def create
     if auction.payment_confirmed? || auction.c2_paid?
       Object.const_get("C2StatusPresenter::#{c2_status}").new(auction: auction)
+    elsif work_in_progress?
```

Comment:
the other benefit to a `work_in_progress` addition to the `status` enum is that we wouldn't need this conditional, we could just use ` Object.const_get("AdminAuctionStatusPresenter::#{status}").new(auction: auction)` below